test/functional/feature_notifications.py

```diff
@@ -32,6 +33,16 @@ def setup_network(self):
                             "-walletnotify=echo > {}".format(os.path.join(self.walletnotify_dir, '%s'))]]
         super().setup_network()
 
+    def test_importmulti(self, updateexisting, count):
+        shutil.rmtree(self.walletnotify_dir)
+        os.mkdir(self.walletnotify_dir)
+
+        addr = self.nodes[0].getnewaddress()
+        self.nodes[0].generatetoaddress(1, addr)
+
+        self.nodes[1].importmulti([{"scriptPubKey": {"address": addr}, "timestamp": "now"}], {"updateexisting": updateexisting})
+        wait_until(lambda: len(os.listdir(self.walletnotify_dir)) == count, timeout=10)
```

Comment:
Comment suggestion
```diff
+        # Wait at most 10 seconds for expected number of files predicate to be true.
         wait_until(lambda: len(os.listdir(self.walletnotify_dir)) == count, timeout=10)
```

Reply:
Code is pretty much self explanatory?

Reply:
True. My thought was to highlight for reviewers unfamiliar with `wait_until` that it can be a form of test assertion but it's definitely optional.